readme.md

```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://MykhailoLoniak.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://MykhailoLoniak.github.io/layout_moyo-header/report/html_report/)
```

Comment:
Add these links to the PR description

Reply:
They were added earlier. I have these links active
![image](https://github.com/mate-academy/layout_moyo-header/assets/139628544/bb991da7-9ce7-4aa5-ba6c-6cf1f11625c6)